dd-java-agent/instrumentation/jax-rs-annotations/src/main/java/datadog/trace/instrumentation/jaxrs/JaxRsAnnotationsInstrumentation.java

```diff
@@ -63,7 +65,7 @@ public JaxRsAnnotationsInstrumentation() {
   public static class JaxRsAnnotationsAdvice {
 
     @Advice.OnMethodEnter(suppress = Throwable.class)
-    public static Scope nameSpan(@Advice.Origin final Method method) {
+    public static List<Scope> nameSpan(@Advice.Origin final Method method) {
```

Comment:
You should return a `Stack<Scope>` instead to clearly indicate that this is a stack that needs to be popped in order to close the scopes.